### PR TITLE
Migrate Packer script to public repo

### PR DESCRIPTION
**Pull request by mjasion**

### CODE REVIEW

File: packer/studio-selfhosted.pkr.hcl [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+locals {
+  aws_tags = {
+    ManagedBy   = "packer"
+    Name        = var.image_name
+    Environment = "prod"
+    BuildDate   = "{{isotime `2006-01-02`}}"
+  }
+
+  aws_release_regions = [
```
Comment (jesper7) [inline packer/studio-selfhosted.pkr.hcl:27]: Thoughts on enabling additional regions?

File: packer/README.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# Studio Selfhosted AMI instance
+
+This AMI contains K3s with preconfigured Ingress Nginx.
```
Comment (jesper7) [inline packer/README.md:3]:
```suggestion
This AMI contains K3s along with a preconfigured ingress-nginx reverse proxy. 
```

File: packer/README.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# Studio Selfhosted AMI instance
+
+This AMI contains K3s with preconfigured Ingress Nginx. 
+
+## Studio Installation
+To deploy Studio please:
+1. Run instance with AMI, 
+2. Open on Security Group ports: 22, 80, 443(if https is needed),
+3. Login though ssh to the instance,
+3. Proceed with the instructions of [Studio helm chart](https://github.com/iterative/helm-charts), how tu configure and install the Studio.
```
Comment (jesper7) [inline packer/README.md:10]:
```suggestion
## Installation
1. Create an EC2 instance with the AMI 
2. In the Security Group, allow the following ports: 22, 80, and 443 (if TLS is needed)
3. Log in to the instance with SSH
4. Proceed with the [Studio Helm chart instructions](https://github.com/iterative/helm-charts) on how to configure and install Studio.
```

File: packer/README.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+# Studio Selfhosted AMI instance
```
Comment (jesper7) [inline packer/README.md:1]:
```suggestion
# Studio Selfhosted AMI
```

File: packer/DEVELOPMENT.md [hunk 5]
```diff
@@ -0,0 +1,7 @@
+# studio-selfhosted-packer
+Packer AMI definition for Studio Selfhosted
```
Comment (jesper7) [inline packer/DEVELOPMENT.md:2]:
```suggestion
To build the AMI and publish it to the AWS marketplace, use the following commands:
```

File: packer/DEVELOPMENT.md [hunk 6]
```diff
@@ -0,0 +1,7 @@
+# studio-selfhosted-packer
```
Comment (jesper7) [inline packer/DEVELOPMENT.md:1]:
```suggestion
# Development
```

File: packer/studio-selfhosted.pkr.hcl [hunk 7]
```diff
@@ -0,0 +1,129 @@
...
+    inline = ["/usr/bin/cloud-init status --wait"]
+  }
+
+  # Install script running as 'root'
+  provisioner "shell" {
+    script          = "${path.root}/setup_root.sh"
+    execute_command = "chmod +x {{ .Path }}; {{ .Vars }} sudo bash {{ .Path }}"
+  }
+
+  # Configure 'ubuntu' user to manage Studio
```
Comment (jesper7) [inline packer/studio-selfhosted.pkr.hcl:125]:
```suggestion
# Install script running as 'ubuntu'
```

**Review by jesper7 — CHANGES_REQUESTED** [review]
:+1: 
Few minor notes

File: packer/README.md (hunk 3 above)
Comment (mjasion) [reply packer/README.md:10]: @jesper7 How about sharing the link to Notion?

File: packer/studio-selfhosted.pkr.hcl (hunk 1 above)
Comment (mjasion) [reply packer/studio-selfhosted.pkr.hcl:27]: I think we have it blocked somewhere - probably we need to enable other regions in console first..

File: packer/README.md (hunk 3 above)
Comment (jesper7) [reply packer/README.md:10]: Good idea. Let's do it

File: packer/studio-selfhosted.pkr.hcl (hunk 1 above)
Comment (jesper7) [reply packer/studio-selfhosted.pkr.hcl:27]: I think you're right. I see that the ones we have enabled are AWS defaults, so let's just keep them

File: packer/studio-selfhosted.pkr.hcl [hunk 8]
```diff
@@ -0,0 +1,129 @@
...
+    "eu-west-1",
+    "eu-west-2",
+    "eu-west-3",
+    #    "me-central-1",
+    #    "me-south-1",
+    #    "sa-east-1",
+    "us-east-1",
+    "us-east-2",
+    "us-west-1",
+    "us-west-2",
```
Comment (jesper7) [inline packer/studio-selfhosted.pkr.hcl:54]:
```suggestion
    #    "af-south-1",
    #    "ap-east-1",
    "ap-northeast-1",
    "ap-northeast-2",
    "ap-northeast-3",
    "ap-south-1",
    #    "ap-south-2",
    "ap-southeast-1",
    "ap-southeast-2",
    #    "ap-southeast-3",
    #    "ap-southeast-4",
    "ca-central-1",
    "eu-central-1",
    #    "eu-central-2",
    "eu-north-1",
    "eu-south-1",
    #    "eu-south-2",
    "eu-west-1",
    "eu-west-2",
    "eu-west-3",
    #    "me-central-1",
    "me-south-1",
    "sa-east-1",
    "us-east-1",
    "us-east-2",
    "us-west-1",
    "us-west-2",
```

Added the remaining regions. I've verified that we have them in our AWS account

Comment (mjasion) [reply packer/studio-selfhosted.pkr.hcl:54]: I enabled all regions 🎉

File: packer/README.md (hunk 3 above)
Comment (mjasion) [reply packer/README.md]: I have added a link to new docs :-) (I know they are not ready but they will be :P)

**Review by jesper7 — APPROVED** [review]

**Pull request merged**